src/deepsparse/transformers/eval_downstream.py

```diff
@@ -462,6 +521,77 @@ def _split_train_val(train_dataset, val_ratio, seed=42):
     return train_ds, val_ds
 
 
+def _process_concatenated_datasets(dataset_name, model_path, max_sequence_length, kwargs):
```

Comment:
can we move these to a helpers file? eval_downstream is getting a bit long so trying to avoid too much task specific logic

Reply:
Done. To limit the scope of the PR to the PPL evaluation, mI moved only the helper functions defined in this PR to utils/eval_helpers. It's probably best to move other helper functions in a future PR.